rest-api-spec/src/main/resources/rest-api-spec/test/indices.put_mapping/20_mix_typeless_typeful.yml

```diff
@@ -41,7 +38,6 @@
  - do:
       catch: /the final mapping would have more than 1 type/
       indices.put_mapping:
-          include_type_name: true
           index: index
           type: some_other_type
```

Comment:
The presence of `type` seems to be not right, it should not be used, right?

Reply:
@reta : Thanks for pointing this out. This also explains the existing test failures.